pkg/actions/irsa/update_test.go

```diff
@@ -83,5 +98,60 @@ var _ = Describe("Update", func() {
 				Expect(fakeStackManager.UpdateStackCallCount()).To(Equal(0))
 			})
 		})
+
+		When("the role does not exist for a service account", func() {
+			It("errors", func() {
+				fakeStackManager.ListStacksMatchingReturns([]*cloudformation.Stack{
+					{
+						StackName: aws.String("eksctl-my-cluster-addon-iamserviceaccount-default-test-sa"),
+						Outputs: []*cloudformation.Output{
+							{
+								OutputKey:   aws.String(outputs.IAMServiceAccountRoleName),
+								OutputValue: aws.String("arn:aws:iam::123456789111:role/test-role"),
+							},
+						},
+					},
+				}, nil)
+
+				err := irsaManager.UpdateIAMServiceAccounts(serviceAccount, false)
+				Expect(err).NotTo(HaveOccurred())
+			})
+		})
```

Comment:
`Expect(err).NotTo(HaveOccurred())` & `It("errors", func() {` :thinking:

Reply:
Uh, some bad copy pasta going on here 🤔 Well spotted. :D I'll take a look.

Reply:
ah yes, I was fixing the previous tests. This is an obsolete one I didn't delete.

Reply:
NOooo I know now haha. This shouldn't be NotTo Have Occurred.

Reply:
Well spotted, I fixed and extended it by one more. :)